Doc/library/io.rst

```diff
@@ -306,8 +306,8 @@ I/O Base Classes
 
 .. class:: IOBase
 
-   The abstract base class for all I/O classes, acting on streams of bytes.
-   There is no public constructor.
+   The abstract base class for all I/O classes. The constructor accepts no
+   arguments.
```

Comment:
The suggested replacement is as wrong as the original.  Better to just delete it here and everywhere else.
```suggestion
   The abstract base class for all I/O classes.
```
I will explain on the issue.

Reply:
Thanks for the catch - would you prefer to wait for a concurrence by another core dev or apply the changes now?

Reply:
> Issue title is too long in any case, but should be changed in meaning also.

Also, the titles of PRs should be in the form "Reword io docstring..." rather than "Reworded io docstring..."
(see https://devguide.python.org/pullrequest/#making-good-commits).

Reply:
I changed the title.

Reply:
Please change the PR to match.

Reply:
Done, and thanks Irit for the link.

Reply:
As an aside, is it standard to give credit to the patch author on the issue or to whoever suggested the wording with a line like `Co-author: <email>` in the PR description, or is that not done as it then wouldn't give credit to the commenters, who in some sense also contributed in arriving at a final wording?

Reply:
If you are converting an old patch to GitHub then the original author should be credited.

Comments on PRs usually not.

There could be a situation where the comments were major, when the commenter is not a core dev, and you decide to offer to credit them as co-authors. I did that once, when someone put a lot of effort into a review.

Core devs are supposed to review contributions and mentor contributors so there is no need.

Reply:
Yes, please give credit to the original patch author via `Co-authored-by: Author Name <email_address>`

See:

* https://devguide.python.org/pullrequest/?highlight=Co-authored-by#converting-an-existing-patch-from-b-p-o-to-github
* https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Reply:
Thanks, and what if the wording was from a variety of discussions on an issue, say [here](https://bugs.python.org/issue12165)? We have Terry who wrote the original wording, then Eryk who elaborated, and I grabbed the latter's wording and applied it to a PR. Would I credit them both via `Co-authored-by`, or just Terry who wrote the majority of it, or is it that conventionally no co-author credit is needed if no patch is prepared?

Reply:
You wouldn’t (1) they are core devs (2) even if they weren’t, commenting in a discussion is not quite authorship.

Reply:
I think it's needed to put `Co-authored-by: Author Name <email_address>` (with the real email address so Git can match it up) in the git commit message, not just the PR body?

Reply:
I'm not too sure what method is commonly used to get a user's public github email address, but I found a [public commit](https://github.com/vadmium/usenet-downloader/commit/940f46210d43102bc9f2c998a3ac78970ab4e608.patch) and there the email was not proper, so I only assumed that they didn't want a proper one written in public to avoid bot scraping.